pom.xml

```diff
@@ -426,6 +425,13 @@
                      <exclude>**/Abstract*.java</exclude>
                   </excludes>
                </configuration>
+               <dependencies>
+                  <dependency>
+                     <groupId>org.aspectj</groupId>
```

Comment:
Not sure why this is required. Tests are all passing without `aspectjweaver`

Reply:
Thanks for the hint, just removed it.